modules/example/src/smithy4s/example/DefaultTest.scala

```diff
@@ -0,0 +1,18 @@
+package smithy4s.example
+
+import smithy4s.schema.Schema._
+
+case class DefaultTest(one: Int, two: String, three: List[String])
```

Comment:
Guessing we are going to want this to be:

```suggestion
case class DefaultTest(one: Int = 1, two: String = "test", three: List[String] = List.empty)
```

If so, I'll make the changes to facilitate that

Reply:
Yeah. But also we need to make sure that the ordering of the fields goes as such : 

1. fully required fields first 
2. required fields with default second 
3. optional fields last. 

The fields are already getting sorted to ensure that optional fields come AFTER required ones. Just change the ordering there .